webapp/packages/core-blocks/src/Menu/useContextMenuPosition.ts

```diff
@@ -10,17 +10,15 @@ import { observable } from 'mobx';
 import { useObservableRef } from '../useObservableRef.js';
 
 export interface IContextMenuPosition {
-  x: number;
-  y: number;
-}
-
-export interface IMouseContextMenu {
-  position: IContextMenuPosition | null;
+  position: {
+    x: number;
+    y: number;
+  } | null;
```

Comment:
I don't personally like inlining public types cuz it's impossible to use them if needed only with `SomeType['someKey']`